src/Typeahead.vue

```diff
@@ -13,9 +11,9 @@
       @keydown.esc="reset"
       @blur="showDropdown = false"
     />
-    <ul class="dropdown-menu" ref="dropdown">
-      <li v-for="(item, index) in items" v-bind:class="{'active': isActive(index)}">
-        <a @mousedown.prevent="hit" @mousemove="setActive(index)">
+    <ul v-bind:class="{'show':showDropdown}" class="dropdown-menu search-dropdown-menu" ref="dropdown" >
```

Comment:
1. It is not necessary to type `v-bind:class="..."`, a colon will do --> `:class="..."`
1. Remove the empty space at the end --> "... ref="dropdown">